scene/gui/split_container.cpp

```diff
@@ -181,7 +181,7 @@ void SplitContainer::_notification(int p_what) {
 			if (dragger_visibility != DRAGGER_VISIBLE)
 				return;
 
-			int sep = dragger_visibility != DRAGGER_HIDDEN_COLLAPSED ? get_constant("separation") : 0;
+			int sep = get_constant("separation");
```

Comment:
`dragger_visibility` could still be `DRAGGER_HIDDEN`, no? And again, we should not assume that enums will never change.